array-from.js

```diff
@@ -67,6 +67,7 @@ if (!Array.from) {
 				}
 				defineProperty(A, k, {
 					'value': mappedValue,
+					'writeable': true,
```

Comment:
- it's spelled `writable`. 
 - can you please add a test that would fail without this change?